indico/modules/rb/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -890,14 +894,17 @@ class RoomEditModal extends React.Component {
             reservationsNeedConfirmation: false,
             capacity: null,
             attributes: roomAttributes,
+            protectionMode: 'public',
             ...roomEquipment,
             ...roomAvailability,
         } : {
             ...room,
+            protectionMode: room.isPublic ? 'public' : 'protected',
```

Comment:
just add the `protection_mode` in the marshmallow schema, then no extra handling is needed here

Reply:
Yeah, I wasn't sure whether to use already exising `isPublic` or add `protection_mode` to the schema